vcd/resource_vcd_independent_disk.go

```diff
@@ -207,3 +340,92 @@ func validateBusSubType(v interface{}, k string) (warnings []string, errors []er
 	}
 	return
 }
+
+var helpError = fmt.Errorf(`resource id must be specified in one of these formats:
```

Comment:
This variable is visible in all the package. Better name it `helpDiskError`.

Reply:
updated